WebJobs.Script.proj

```diff
@@ -107,7 +108,11 @@
              BuildInParallel="$(BuildInParallel)"/>
   </Target>
 
-  <Target Name="PackageScriptHost" DependsOnTargets="Build">
+  <Target Name="DownloadRuntimeNugetClient">
+    <DownloadNuGet OutputFileName="$(FunctionsSiteExtensionPath)\bin\tools\nuget.exe" MinimumVersion="3.3.0" DownloadUrl="https://facavalcspn.blob.core.windows.net/public/nuget.exe"  />
```

Comment:
Did you mean to leave this pointing to your blob storage? Is there a way that others can update the nuget.exe there if we need to?